fvm/evm/types/call.go

```diff
@@ -21,6 +21,7 @@ const (
 	TransferCallSubType = byte(3)
 	DeployCallSubType   = byte(4)
 	ContractCallSubType = byte(5)
+	DryRunSubType       = byte(6)
```

Comment:
Do we need a new subtype, doesn't contract call makes sense here?

Reply:
Also we don't emit event for dryruns.

Reply:
It can be more than just a call, it can be a deployment for example

Reply:
It can also be just a simple value transfer ^.